libcperciva/network/network.h

```diff
@@ -40,13 +40,25 @@ void * network_connect(struct sock_addr * const *,
     int (*)(void *, int), void *);
 
 /**
- * network_connect_timeo(sas, timeo, callback, cookie):
- * Behave as network_connect(), but wait a duration of at most ${timeo} for
- * each address which is being attempted.
+ * network_connect_bind(sas, sa_b, callback, cookie):
+ * Iterate through the addresses in ${sas}, attempting to create and connect
+ * a non-blocking socket after binding to ${sa_b}.  Once connected, invoke
+ * ${callback}(${cookie}, s) where s is the connected socket; upon fatal error
+ * or if there are no addresses remaining to attempt, invoke
+ * ${callback}(${cookie}, -1).  Return a cookie which can be passed to
+ * network_connect_cancel() in order to cancel the connection attempt.
  */
-void * network_connect_timeo(struct sock_addr * const *, const struct timeval *,
+void * network_connect_bind(struct sock_addr * const *, struct sock_addr const *,
```

Comment:
I normally spell this as `const struct sock_addr *` rather than `struct sock_addr const *`.  (The declarations are equivalent so this is just a style issue.)

Reply:
Fixed.